### ISSUE TITLE
ADAG Naming

### ISSUE THREAD

**Issue by JoeriHermans** [tmulc18/Distributed-TensorFlow-Guide#7]
Hi,

This is a very nice write-up! I'm the author of ADAG, I recently changed the name of the optimizer into AGN (Accumulated Gradient Normalization). Which is described in full here https://arxiv.org/abs/1710.02368

Just to keep everything consistent :) 

Joeri

**Comment by tmulc18** [comment]
Hi @JoeriHermans, 

Thank you, and thanks for letting me know about the change!  I've made the updates :)  Recently, I recommended someone to read your thesis posted on your GitHub.  Is that up-to-date?  I thought I saw a few references to ADAG instead of AGN, but maybe they were referring to different topics.

Tommy
